src/components/TxDetailsCard.js

```diff
@@ -0,0 +1,106 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+// @flow
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, Platform, StyleSheet, TouchableNativeFeedback, TouchableOpacity } from 'react-native';
+import colors from '../colors';
+import Card from './Card';
+import AccountIcon from './AccountIcon';
+
+const WEI_IN_ETH = 1000000000000000000;
+
+export default class TxDetailsCard extends React.Component<{
+  value: string,
+  recipient: string,
+  networkId: number
+}> {
+  static propTypes = {
+    value: PropTypes.string.isRequired,
+    gas: PropTypes.string.isRequired,
+    gasPrice: PropTypes.string.isRequired,
+    style: View.propTypes.style
+  };
+
+  render() {
+    const { value, gas, gasPrice, recipient, networkId, style } = this.props;
+
+    return (
+      <View style={[styles.body, style]}>
+        <Text style={styles.titleText}>You are about to send</Text>
+        <Amount style={{ marginTop: 10 }} value={value} gas={gas} gasPrice={gasPrice} />
+      </View>
+    );
+  }
+}
+
+function Amount({ style, value, gas, gasPrice }) {
+  const fee = parseInt(gas) * parseInt(gasPrice) / WEI_IN_ETH;
+  return (
+    <View style={[{ justifyContent: 'center', alignItems: 'center' }, style]}>
+      <View>
+        <View style={{ padding: 5, paddingVertical: 2, backgroundColor: colors.bg }}>
```

Comment:
In general it's better to put style in a `StyleSheet.create`, because like this it will create a new JS object on every render.